app/services/imports/lettings_logs_field_import_service.rb

```diff
@@ -36,6 +38,29 @@ def update_offered(xml_doc)
       end
     end
 
+    def update_creation_method(xml_doc)
+      old_id = meta_field_value(xml_doc, "document-id")
+      log = LettingsLog.find_by(old_id:)
+
+      if log
+        upload_method = meta_field_value(xml_doc, "upload-method")
+
+        case upload_method
+        when "Manual Entry"
+          @logger.info "lettings log with old id #{old_id} entered manually, no need for update"
+        when "Bulk Upload"
+          if log.creation_method_bulk_upload?
+            @logger.info "lettings log #{log.id} creation method already set to bulk upload, no need for update"
+          else
+            log.creation_method_bulk_upload!
+            @logger.info "lettings log #{log.id} creation method set to bulk upload"
+          end
```

Comment:
Would there be a situation where the log already has the creation method set? Wouldn't just having the else part of the statement achieve the same result?

Reply:
oh I think I see what you mean, if the old log is bulk upload, why not just set it to bulk upload in any case and have fewer lines?

for that I was following the model of the existing methods in the field import services, which seemed to check whether the field was already set correctly and log that if so